js/admin/notifications.js

```diff
@@ -79,11 +78,10 @@ function renderCustomerNotification(value) {
 }
 
 function renderMessageNotification(value) {
-  const query = `tab=AdminCustomerThreads&token=${token_admin_customer_threads}&viewcustomer_thread&id_customer_thread=${value.id_customer_thread}`;
   const company = value.company !== '' ? ` (${value.company})` : '';
   return `
-    <a class="notif" href="${baseAdminDir}index.php?${query}">
-      <span class="message-notification-status ${value.status}">
+    <a class="notif" href="${value.customer_thread_view_url}">
+      <span class="message-notification-status customer_thread_view_url{value.status}">
```

Comment:
I'm not sure to understand here 🤔
```suggestion
      <span class="message-notification-status ${value.status}">
```

Reply:
Fixed @atomiix